tools/deploy_translatable_strings.sh

```diff
@@ -89,6 +89,8 @@ cp "$SOURCE_REPO_ROOT/qiskit_pkg/setup.py" "${TARGET_REPO_ROOT}"
 cat "$SOURCE_REPO_ROOT/requirements-dev.txt" "$SOURCE_REPO_ROOT/requirements-optional.txt" \
     > "${TARGET_REPO_ROOT}/requirements-dev.txt"
 cp "$SOURCE_REPO_ROOT/constraints.txt" "${TARGET_REPO_ROOT}"
+# Add commit hash to be able to run the build with the commit hash before the actual release
+echo $QISKIT_COMMIT > "${TARGET_REPO_ROOT}/qiskit-commit-hash"
```

Comment:
As it stands, we haven't actually defined this `QISKIT_COMMIT` environment variable yet in the file.  We'll need to either ask `git` for it (from the source repo), or feed it into this file from the CI pipelines.

Reply:
Oh I think it should be `GITHUB_SHA` instead